src/main/java/com/juniorstart/juniorstart/model/User.java

```diff
@@ -9,13 +9,16 @@
 import org.hibernate.annotations.NaturalId;
 
 import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.UUID;
 
 @Entity
 @Table(name = "users", uniqueConstraints = {
         @UniqueConstraint(columnNames = "email")
 })
-@Data
+@Setter
+@Getter
```

Comment:
Dlaczego usunięte @Data i zastąpiono to @Setter i @Getter? Jest jakiś powód konkretny?

Reply:
+1